### PR TITLE
guest info adjustmets

### PR DESCRIPTION
**Pull request by KenPrz**
**minor changes for guest info routing**

1. added routes for guest info module in web.php
2. changed file extensions from html to blade.php
3. removed unnecessary tags in frontdesk
4. added button routing in frontedsk dashboard (reservation).

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -49,3 +49,20 @@
...
+    return view('guest-information/Reservation');
+});
+
+Route::get('/reservation-list', function () {
+    return view('guest-information/Reservation List');
+});
+
+Route::get('/blacklist', function () {
+    return view('guest-information/Blacklist');
+});
```
Comment (FN-FAL113) [inline routes/web.php]: add ka module prefix, '```/guestinfo```/reservation', apply for all

Comment (FN-FAL113) [reply routes/web.php]: para di mag conflict soon enough

**Review by FN-FAL113 — CHANGES_REQUESTED** [review]

File: routes/web.php (hunk 1 above)
Comment (KenPrz) [reply routes/web.php]: noted

**Comment by KenPrz** [comment]
added module prefix on guest info module

**Review by FN-FAL113 — APPROVED** [review]

**Review by Teds30 — APPROVED** [review]

**Pull request merged**